R/EXADBI-internal.R

```diff
@@ -33,11 +34,10 @@ grep_schema <- function(stmt , statement = TRUE) {
 
 # Changes an identifier into uppercase, except for it is quoted.
 #
-# name EXAupper
 # param identifier A character vector containing one or many (table, schema, column,...) identifiers.
 # return A character vector containing one or many processed identifiers.
-# export
-EXAupper <- function(identifier) {
+# @keywords internal
+.EXAupper <- function(identifier) {
```

Comment:
The dot indicates functions as internal, and R CMD check won't complain about missing documentation!